README.md

```diff
@@ -44,7 +44,7 @@ Loading this module through a script tag will make it's exports available as `Li
 
 ```js
 import { createLibp2p } from 'libp2p'
-import { Noise } from '@chainsafe/libp2p-noise'
+import { noise } from "@chainsafe/libp2p-noise";
```

Comment:
Please use single quotes and remove the semi-colons otherwise our linter will fail